README.rst

```diff
@@ -36,3 +36,21 @@ Initial default test login credentials are:
 
 **Password:** password 
 
+Testing
+-------
+
+To run the webserver locally, first run the following commands to insert a user with administrative access::
+
+    \c ag_test
+
+    INSERT INTO ag.labadmin_users (email, password) VALUES ('master', '$2a$10$2.6Y9HmBqUFmSvKCjWmBte70WF.zd3h4VqbhLMQK1xP67Aj3rei86');
+    
+    INSERT INTO ag.labadmin_users_access (access_id, email) VALUES (7, 'master');
+    
+Then you should be able to login with the following credentials
+
+**User:** master
+
+**Password:** password 
+
+Note that whenever a unittest is run, the user `test` is granted admin rights, which is reset once the unittest is completed if `mock_admin_login` is called.
```

Comment:
To me, Jamie's description seems to be accurate, since tearDown() alters the 'test' user access level:
`    def tearDown(self):
        BaseHandler.get_current_user = self.orig_func
        # Remove all access privileges user may hve been given by a test
        db.alter_access_levels('test', [])
        super(TestHandlerBase, self).tearDown()
`
Which means that a manual change of the access rights for this user are overwritten by any unit test.